packages/graphback-core/src/runtime/interfaces.ts

```diff
@@ -14,9 +14,7 @@ export interface GraphbackServiceConfigMap {
  * GraphQL context interface according to Graphback runtime layer format
  */
 export interface GraphbackContext {
-  graphback: {
-    services: GraphbackServiceConfigMap
-  }
+  services: GraphbackServiceConfigMap
```

Comment:
`services` is too general and it is likely that the `context` could contain a key with this name. What do you think of `graphbackServices`?

Reply:
I was thinking the same, and would like to use `models`, but then it would also need to be changed from `services` to `models` in `GraphbackAPI` (see API reference)[https://graphback.dev/docs/next/api/graphback/interfaces/_buildgraphbackapi_.graphbackapi#services] for consistency, but here it might gert confusing as the user might think that models means actual model GraphQL files or something.

I kept it as `services` then because we originally tried the approach of being ultra flexible and fitting into any use case without interfering with some user defined code - but this lead us down the road of having really unfriendly APIs which we are rolling back on now.

So in essence, a prerequisite of using Graphback could be that our services will be called services in the context, you will need to update your code accordingly.

Alternatively, we could just use `graphback` and have no `services` key:

```ts
context.graphback.Note.findBy(...);
```

I like conciseness in a user API, don't mind long variables internally, but love to make things simple for the user.

Reply:
> I was thinking the same, and would like to use `models`, but then it would also need to be changed from `services` to `models` in `GraphbackAPI` (see API reference)[https://graphback.dev/docs/next/api/graphback/interfaces/_buildgraphbackapi_.graphbackapi#services] for consistency, but here it might gert confusing as the user might think that models means actual model GraphQL files or something.
> 
> I kept it as `services` then because we originally tried the approach of being ultra flexible and fitting into any use case without interfering with some user defined code - but this lead us down the road of having really unfriendly APIs which we are rolling back on now.
> 
> So in essence, a prerequisite of using Graphback could be that our services will be called services in the context, you will need to update your code accordingly.

This may work fine if user looks set to define custom services, but it maybe not play well if the user has another framework that plays with the context and uses `services` as a key. 

> Alternatively, we could just use `graphback` and have no `services` key:
> 
> ```ts
> context.graphback.Note.findBy(...);
> ```
> 
> I like conciseness in a user API, don't mind long variables internally, but love to make things simple for the user.

I like the alternative suggestion.

Reply:
Same, I will go with that.

Reply:
Thanks.